extensions/warp-ipfs/tests/direct.rs

```diff
@@ -313,11 +314,11 @@ mod test {
         let message_b = tokio::time::timeout(Duration::from_secs(60), async {
             loop {
                 if let Some(MessageEventKind::MessageReceived {
-                    conversation_id,
+                    conversation_id: _,
```

Comment:
I think you can just skip everything with `..` here.
```suggestion
                    ..
```

Reply:
left intentionally (since i didnt complete the whole test but meant to compare conversation id but do keep in mind this pr is still in a draft state and just putting the concept out there)

Reply:
sure. I noticed half way through both PRs that they're still drafts. :)